### PR TITLE
Option to disable terminator

### PR DESCRIPTION
**Pull request by chri87d9**
I am new to github, so please bare over with me. I read that i was supposed to rebase the changes and that is was i have tried to do. I am sorry for the messy commits, i accidentally addressed multiple things in the branch, and ad to clean it up. 

- I added an option to disable the terminator under: preferences/modules/map view/extra contents, this is in relation to #3

### CODE REVIEW

File: src/gtk-sat-map.c [hunk 1]
```diff
@@ -165,7 +165,7 @@ static void gtk_sat_map_init(GtkSatMap * satmap)
     satmap->height = 0;
     satmap->refresh = 0;
     satmap->counter = 0;
-    satmap->show_terminator = FALSE;
```
Comment (csete) [inline src/gtk-sat-map.c:168]: I think the default should stay FALSE at least until the feature is complete and well tested.

File: src/sat-cfg.c [hunk 2]
```diff
@@ -118,6 +118,7 @@ sat_cfg_bool_t  sat_cfg_bool[SAT_CFG_BOOL_NUM] = {
     {"GLOBAL", "MOD_WIN_POS", FALSE},
     {"GLOBAL", "MOD_STATE", FALSE},
     {"MODULES", "RULES_HINT", FALSE},
+    {"MODULES", "MAP_SHOW_TERMINATOR", TRUE},
```
Comment (csete) [inline src/sat-cfg.c:121]: Same as above, the default value should stay FALSE for now.

File: src/sat-pref-map-view.c [hunk 3]
```diff
@@ -36,7 +36,7 @@ static GtkWidget *thumb;
 static gchar   *mapf = NULL;
 
 /* content selectors */
-static GtkWidget *qth, *next, *curs, *grid;
+static GtkWidget *qth,*term, *next, *curs, *grid;
```
Comment (csete) [inline src/sat-pref-map-view.c:39]: Missing space.

**Comment by chri87d9** [comment]
I followed the guide and tried squashing the commits, hope it is better now :)

**Comment by csete** [comment]
The squashing doesn't seem to have worked. Anyhow, I have merged pull request #236 which also addressed the issue with map centering, so this pull request can be closed now.

**Pull request closed without merging**